R/derive_var_dthcaus.R

```diff
@@ -249,13 +267,17 @@ derive_var_dthcaus <- function(dataset,
 dthcaus_source <- function(dataset_name,
                            filter,
                            date,
+                           date_imputation = NULL,
```

Comment:
@tamarasenior as per new partial date refactor rules, this function won't need to allow for date imputation any more and this would be done as pre-processing so you only would expect DT or DTM dates here now. this was explained at the admiral community meeting Friday (check out the recording on Slack in case you missed it)